backend/NycBasics/models.py

```diff
@@ -32,3 +32,26 @@ class parking_model(models.Model):
 class bench_model(models.Model):
     bench_latitude = models.FloatField(null=True)
     bench_longitude = models.FloatField(null=True)
+
+
+class Rating_Review(models.Model):
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    amenity_type = models.CharField(max_length=10, null=False)
+    amenity_id = models.IntegerField(null=False, default=0)
+    # water_id = models.ForeignKey(water_model, on_delete=models.CASCADE)
+    # bench_id = models.ForeignKey(bench_model, on_delete=models.CASCADE)
```

Comment:
why are these lines commented?

Reply:
trying for a different approach , but the current code has good security and functionality.